vhdbuilder/scripts/windows/automate_version_bump.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -euxo pipefail
+
+github_user_name=$2
+source vhdbuilder/scripts/windows/automate_helpers.sh
+
+az login --identity
+
+pr_title="VHDVersion Bumping"
```

Comment:
Is it not pr title? It seems like that it is only used for logging.

Reply:
It's used only for logging. But since in the release note script later, this will be something like `releaseNotes` I have changed it to `pr_purpose`.